cds/modules/records/static/templates/cds_records/video/downloads.html

```diff
@@ -18,7 +18,7 @@ <h3 class="cds-title-section-decoration bt bw-1 pt-10 mb-20">
   </div>
   <!-- Master file -->
   <div class="cds-video-master-file mb-20">
-    <a ng-click="dialogShown = true" class="btn btn-block btn-default">Master video ({{ (record | findMaster).size | bytesToHumanReadable }})</a>
+    <a ng-click="dialogShown = true" class="btn btn-block btn-default">Master video ({{ (record | findMaster).size | bytesToHumanReadable}}) {{record | findResolution}}</a>
```

Comment:
It is probably better to you use again `findMaster`:
` (record | findMaster) | findResolution`